core/helper.rb

```diff
@@ -130,9 +131,12 @@ def destroy_config(config_name)
   if Dir.exist? config_name
     unless get_provider(config_name) == MDBCI
       root_dir = Dir.pwd
-      Dir.chdir config_name
-      execute_bash('vagrant destroy -f')
-      Dir.chdir root_dir
+      begin
```

Comment:
This block repeats several times below. 
Probably, it should be extracted in a separated function - executeBashInConfigDirSafely?